media/css/pocket/components/home.scss

```diff
@@ -4,22 +4,22 @@
 
 @use 'sass:color';
 
+$image-path: '/media/protocol/img';
+
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 @import '~@mozilla-protocol/core/protocol/css/base/elements/containers';
 @import '~@mozilla-protocol/core/protocol/css/components/button';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-pocket';
 @import '../utils/variables';
 @import '../includes/platform-nav';
 @import '../includes/platform-footer';
 
-$image-path: '/media/img/pocket';
```

Comment:
removed as this wasn't used in the file and conflicts with expected protocol variable